services/offer.js

```diff
@@ -52,7 +53,13 @@ const offerService = {
   updateOffer: async (id, currentUserId, updateData) => {
     const filteredUpdateData = filterAllowedFields(updateData, allowedOfferFieldsForUpdate)
 
-    const offer = await Offer.findById(id)
+    const offer = await Offer.findById(id).exec()
+    console.log('Current User ID:', currentUserId)
```

Comment:
Would you mind considering delete console logs?

Reply:
Done! Thank you for your attention